theme/theme.scss

```diff
@@ -29,3 +29,14 @@
 [class*="ck-heading_heading"] {
 	font-weight: bold;
 }
+
+// Resize dropdown's button label.
+.ck-dropdown {
+	&.ck-heading-dropdown {
+		.ck-dropdown__button {
+			.ck-button__label {
+				width: 8em;
```

Comment:
👍  – better not make the `width` hardcoded. `min-width` is what we wanted to achieve I think – that dropdowns are not super narrow.